QCVEngine.cabal

```diff
@@ -54,5 +54,5 @@ executable QCVEngine
   build-depends:       base >=4.9, time, split >=0.2, regex-posix >=0.95,
                        binary >=0.8, bitwise >=1.0, network >=2.8, parsec,
                        bytestring >=0.10, filemanip >=0.3,
-                       QuickCheck ==2.12
+                       QuickCheck ==2.12.6.1
```

Comment:
```suggestion
                       QuickCheck >=2.12, QuickCheck <2.13
```
then?

Reply:
Seems like the syntax is `QuickCheck >= 2.12 && < 2.13`. Thanks.